packages/core-mobile/app/services/fcm/FCMService.ts

```diff
@@ -17,6 +19,12 @@ class FCMService {
     return await messaging().getToken()
   }
 
+  listenForTokenRefresh = (dispatch: Dispatch): void => {
+    messaging().onTokenRefresh(_ => {
+      dispatch(onFcmTokenChange)
+    })
+  }
```

Comment:
I don't recommend doing this since FCM service shouldn't know about redux and the specific redux actions. you can still achieve this by passing a callback here instead of dispatch and then call this somewhere else with the `dispatch(onFcmTokenChange)` as the callback.

Reply:
fixed in [Pass callback to listenForTokenRefresh](https://github.com/ava-labs/core-mobile/pull/1659/commits/2eacb68e23248f772a5b3d6d9f3e7741690f819b)